### PR TITLE
Add v1alpha1 nncp population test

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: test/e2e/handler/upgrade_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+					DesiredState: linuxBrUp(bridge1),
+				},
+			}
+			Expect(framework.Global.Client.Create(context.TODO(), &policy, &framework.CleanupOptions{})).To(Succeed(), "should success creating a v1alpha1 nncp")
+		})
+		AfterEach(func() {
+			updateDesiredStateAndWait(linuxBrAbsent(bridge1))
+			resetDesiredStateForNodes()
+		})
+		FIt("should be stored as v1beta1 and end with available state", func() {
```
Comment (phoracek) [inline test/e2e/handler/upgrade_test.go:34]: FFF

File: test/e2e/handler/upgrade_test.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+
+	nmstate "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/shared"
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var _ = Describe("NondeNetworkConfigurationPolicy upgrade", func() {
```
Comment (phoracek) [inline test/e2e/handler/upgrade_test.go:17]: Node

**Review by phoracek — CHANGES_REQUESTED** [review]

**Comment by qinqon** [comment]
Pushed changed to address Petr comments.

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Pull request merged**